keyboards/dz60/keymaps/olivierko/readme.md

```diff
@@ -0,0 +1,19 @@
+# DZ60
+
+### _BL:
+Base layer with american ANSI layout.
```

Comment:
If you terminate a line with two spaces and then a new line, GitHub will turn it into a single-line break. Renders a bit more nicely.

```suggestion
Base layer with american ANSI layout.  
```